### PR TITLE
en: add aws and gcp pitr docs

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/2146
- Other reference link(s):

### CODE REVIEW

File: en/backup-to-gcs-using-br.md [hunk 1]
```diff
@@ -122,6 +131,216 @@ This document provides an example about how to back up the data of the `demo1` T
...
+From the output, you can find the following information for the `Backup` CR named `demo1-full-backup-gcs`. The `COMMITTS` field indicates the time point of the snapshot backup:
+
+```
+NAME                    TYPE   MODE       STATUS     BACKUPPATH                               COMMITTS             ...
+demo1-full-backup-gcs   full   snapshot   Complete   gcs://my-bucket/my-full-backup-folder/   436979621972148225   ...
+```
+
+### Log backup
+
+You can use a `Backup` CR to describe the start and stop of a log backup task and manage the log backup data. In this section, the example shows how to create a `Backup` CR named `demo1-log-backup-s3`. See the following detailed steps.
```
Comment (shichun-0415) [inline en/backup-to-gcs-using-br.md:151]:
```suggestion
You can use a `Backup` CR to describe the start and stop of a log backup task and manage the log backup data. Log backup grants permissions to remote storages in the same way as snapshot backup. In this section, the example shows how to create a `Backup` CR named `demo1-log-backup-s3`. See the following detailed steps.
```

File: en/backup-to-gcs-using-br.md [hunk 2]
```diff
@@ -122,6 +131,216 @@ This document provides an example about how to back up the data of the `demo1` T
...
+kubectl get backup -n backup-test
+```
+
+```
+NAME                       MODE     STATUS    ....
+demo1-log-backup-gcs       log      Stopped   ....
+```
+
+<Tip>
+You can stop log backup by taking the same steps as in [Start log backup](#start-log-backup). The existing `Backup` CR will be updated.
```
Comment (shichun-0415) [inline en/backup-to-gcs-using-br.md:275]:
```suggestion
You can also stop log backup by taking the same steps as in [Start log backup](#start-log-backup). The existing `Backup` CR will be updated.
```

File: en/restore-from-aws-s3-using-br.md [hunk 3]
```diff
@@ -194,9 +208,126 @@ After creating the `Restore` CR, execute the following command to check the rest
...
+
+    If the data to be restored is in Amazon S3, you can grant permissions in three methods. For more information, see [AWS account permissions](grant-permissions-to-remote-storage.md#aws-account-permissions).
+
+    If the data to be restored is in other S3-compatible storage (such as Ceph and MinIO), you can grant permissions by [using AccessKey and SecretKey](grant-permissions-to-remote-storage.md#grant-permissions-by-accesskey-and-secretkey).
+
+### Step 2: Restore the backup data to a TiDB cluster
+
+The example in this section restores the snapshot backup data to the cluster. The specified restoration time point must be between [the time point of snapshot backup](backup-to-aws-s3-using-br.md#view-the-snapshot-backup-status) and the [`Log Checkpoint Ts` of log backup](backup-to-aws-s3-using-br.md#view-the-log-backup-status).
+
+PITR grant permissions to remote storages in the same way as snapshot backup. The example in this section grant permissions by using AccessKey and SecretKey.
```
Comment (shichun-0415) [inline en/restore-from-aws-s3-using-br.md:267]:
```suggestion
PITR grants permissions to remote storages in the same way as snapshot backup. The example in this section grants permissions by using AccessKey and SecretKey.
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Review by WizardXiao — APPROVED** [review]

**Pull request merged**